scompose/project/project.py

```diff
@@ -468,8 +488,7 @@ def _create(
     # Build
```

Comment:
The 'for loop' above prevents some containers from running. Any container that runs indefinitely will block execution of this loop and prevent all subsequent containers from running. Many web service containers operate this way.

I tried to pass command args like "indefinite.sh arg1 >& stdout.txt &" but I get errors related to invalid command args. Unless there's another existing method for backgrounding an indefinite execution script I'm not aware of, this loop ought to provide a new method to allow for this.

Reply:
This probably needs to be addressed separately, it's not in scope for the PR here.  It totally depends on the command that you provide - for most of the use cases I've seen they just are creating the instance without something that would continue running.

Reply:
After this PR is merged, we won't release, but you can open another issue with an example to reproduce and we can work on separately. The scope of this PR is already too big.